Library/Contributions/brew_bash_completion.sh

```diff
@@ -48,6 +48,8 @@ __brewcomp ()
 
     for s in $1; do
         __brewcomp_words_include "$s" && continue
+        # do not auto-complete "instal" abbreviation for "install"
+        [[ $s == instal ]] && continue
         list="$list$s$sep"
```

Comment:
Good catch, thanks! I see the issue you're talking about it. Would you mind explaining that syntax for representing newline though? I looked through the docs, but couldn't find much.